content/01-responsive-web-design/basic-css/add-rounded-corners-with-border-radius.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: bad87fee1348bd9aedf08814
-title: Add Rounded Corners with border-radius
+title: Adicionar cantos arredondados com border-radius
```

Comment:
I think it would be nice if you put "Adicione" instead of "Adicionar". When I read the titles on freeCodeCamp, I have a personal impression to read first verb in imperative tense... as commands lines. It's just a suggestion... 😊

Reply:
@Viniciusalgueiro, could you review this and consider this review topic? Thanks! 😊

Reply:
pronto